src/components/MediaCard.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import { Link } from 'react-router-dom'
+const useStyles = makeStyles({
+    root: {
+        maxWidth: 345,
+        marginBottom: '7px'
+    },
+    media: {
+        height: 140,
+    },
+});
+
+export default function MediaCard(props) {
+
+    const classes = useStyles();
+
+    const addToFavourites = () => {
+        props.addToFavourites(props.result)
+    }
+
+    const deleteFromFavourites = () => {
+        props.deleteFromFavourites(props.favourit)
+    }
+    const goBack = () => {
+        props.history.goBack()
+    }
+
+    return (
+        <Card className={classes.root}>
```

Comment:
When the JSX is too long, consider creating another stateless component that will contain only the UI